content/en/integrations/guide/aws-organizations-setup.md

```diff
@@ -0,0 +1,131 @@
+  tag: "Blog"
+  text: "Monitoring AWS Lambda with Datadog"
+- link: "https://www.datadoghq.com/blog/cloud-security-posture-management/"
+  tag: "Blog"
+  text: "Introducing Datadog Cloud Security Posture Management"
+- link: "https://www.datadoghq.com/blog/datadog-workload-security/"
+  tag: "Blog"
+  text: "Secure your infrastructure in real time with Datadog Cloud Workload Security"
+- link: "https://www.datadoghq.com/blog/announcing-cloud-siem/"
+  tag: "Blog"
+  text: "Announcing Datadog Security Monitoring"
+- link: "https://www.datadoghq.com/blog/tagging-best-practices/#aws"
+  tag: "Blog"
+  text: "Best practices for tagging your infrastructure and applications"
+
+---
+
+## Overview
+
+This guide provides an overview of the process for integrating multiple accounts within an AWS Organization or one or more Organizational Units with Datadog.
+
+The Cloudformation StackSet template provided by Datadog automates the creation of the required IAM role and associated policies in every AWS account under an Organization or Organizational Unit (OU), and configures the accounts within Datadog, eliminating the need for manual setup. Once set up, the integration automatically starts collecting AWS metrics & events for you to start monitoring your infrastructure.
+
+The Datadog Cloudformation StackSet performs the following steps:
+
+1. Deploys the Datadog AWS CloudFormation Stack in every account under an AWS Organization or Organizational Unit.
+2. Automatically creates the necessary IAM role and policies in the target accounts.
+3. Automatically initiates ingestion of AWS CloudWatch metrics and events from the AWS resources in the accounts.
+4. Optionally disables metric collection for the AWS infrastructure (can be disabled for Cloud Cost Management (CCM) or Cloud Security Posture Management (CSPM) only use cases)
+5. Optionally configures Datadog Cloud Security Posture Management to monitor resource misconfigurations in your AWS accounts.
+
+
+Please note that the StackSet does not set up log forwarding in the AWS accounts. To set up logs, follow the steps in the [Log Collection][2] guide.
+
+
+## Prerequisites
+
+Before getting started, ensure you have the following prerequisites:
+
+
+1. **Access to the management account**: Your AWS user needs to be able to access the AWS management account.
+2. **An account administrator has enabled Trusted Access with AWS Organizations**: Refer to [AWS Documentation][3] on how to enable trusted access between StackSets and Organizations to create & deploy stacks using service-managed permissions.
+
+## Setup
+
+To get started, go to the [AWS Integration configuration page][1] in Datadog and click on **Add AWS Account(s)** -> **Add Multiple AWS Accounts** -> **Cloudformation StackSet**.
+
+On the Datadog console, click **Launch CloudFormation StackSet**. This opens the AWS Console and loads a new CloudFormation StackSet. Keep the default choice of “Service-managed permissions” on AWS.  
```

Comment:
I don't think we use the term "Datadog console" anywhere, but I'm not sure. I would just say, "In Datadog, ..." or "In the Datadog AWS Integration configuration page....".

Apply whatever term you want to all instances of this throughout the guide.